media/js/base/stub-attribution.js

```diff
@@ -357,6 +357,56 @@ if (typeof window.Mozilla === 'undefined') {
         }
     };
 
+    /**
+     * Attempts to retrieve the GA4 client and session ID from the dataLayer
```

Comment:
We do not retrieve the session ID here, right?

Reply:
Correct. It's an outdated comment, I will update it. TY!